imm_ptl_core/src/main/java/com/qouteall/immersive_portals/commands/PortalCommand.java

```diff
@@ -1312,6 +1324,34 @@ private static void invokeSetPortalDestination(
             }
         }
     }
+
+    private static void invokeSetPortalLocation(
+            CommandContext<ServerCommandSource> context,
+            Portal portal
+    ) throws CommandSyntaxException {
+        Vec3d pos = Vec3ArgumentType.getVec3(
+                context, "pos"
+                );
+        double posX = pos.getX();
+        double posY = pos.getY() + 1.000;
+        double posZ = pos.getZ();
+        portal.setPos(
+                posX,
+                posY,
+                posZ
+        );
+
+        sendMessage(context, portal.toString());
+
+        if (portal instanceof BreakablePortalEntity) {
+            if (!((BreakablePortalEntity) portal).unbreakable) {
+                sendMessage(context, "You are editing a breakable portal." +
```

Comment:
This warning can be wrapped into another method. You can also remove it because the warning is not mandatory